rocks-db/src/column.rs

```diff
@@ -377,6 +377,18 @@ where
             .collect::<Vec<_>>()
     }
 
+    pub fn pairs_iterator<'a>(
```

Comment:
a handy method that's unused by this PR, it was used for iterations with a previous refactor